tox.ini

```diff
@@ -42,16 +42,10 @@ commands =
 [docs]
 docs_dir = {tox_root}{/}docs
 build_dir = {[docs]docs_dir}{/}_build
-# replace when Sphinx>=7.3 and Python 3.8 is dropped:
-#  -T => --show-traceback
-#  -W => --fail-on-warning
-#  -b => --builder
-#  -v => --verbose
-#  -a => --write-all
-#  -E => --fresh-env
-sphinx_args = -T -W --keep-going --jobs auto
+sphinx_args = --show-traceback --fail-on-warning --keep-going --jobs auto
 
 [testenv:docs{,-lint,-all,-live,-live-src}]
+base_python = py312
 package = wheel
 wheel_build_env = .pkg
 change_dir = docs
```

Comment:
I realized in #498 I forgot to remove `change_dir` when I transitioned all this to use absolute paths; given this is already touching all this, seems most prudent to remove it here.